### PR TITLE
fix: product switch trunc

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#402

## Description
Making the title encapsulate on first line if it has a subtitle or go to the second line and encapsulate there if it has no subtitle.

### Before:
Screen Shot 2019-10-30 at 2 04 26 PM
Screen Shot 2019-10-30 at 2 04 00 PM
Screen Shot 2019-10-30 at 2 03 36 PM

### After:
Screen Shot 2019-10-30 at 2 05 03 PM
Screen Shot 2019-10-30 at 2 04 55 PM
Screen Shot 2019-10-30 at 2 04 44 PM

### CODE REVIEW

File: docs/pages/components/product-switch.md [hunk 1]
```diff
@@ -28,93 +28,93 @@ Product Switch provides a role based access to all products or LoBs. It shows on
             <div class="fd-popover__body fd-popover__body--right" aria-hidden="true" id="product-switch-body">
                 <div class="fd-product-switch__body">
                     <ul class="fd-product-switch__list">
-                        <li class="fd-product-switch__item" tabindex="0">
+                        <li class="fd-product-switch__item">
```
Comment (jbadan) [inline docs/pages/components/product-switch.md]: Did you mean to get rid of `tabindex='0'` on all of these?

File: src/product-switch.scss [hunk 2]
```diff
@@ -48,7 +48,6 @@ $block: #{$fd-namespace}-product-switch;
     overflow-y: auto;
     padding: 1.5rem 0.5rem 1rem 1rem;
     background-color: var(--sapList_Background);
-    border-radius: var(--sapElement_BorderCornerRadius);
```
Comment (InnaAtanasova) [inline src/product-switch.scss]: Why did you remove the border-radius?

File: src/product-switch.scss [hunk 3]
```diff
@@ -88,10 +86,28 @@ $block: #{$fd-namespace}-product-switch;
...
+
+      .#{$block}__title {
+        &::after,
+        &::before {
+          background-color: var(--sapTile_Background);
+        }
+
+        &::before {
+          left: 24.7em;
+          width: 1em;
```
Comment (InnaAtanasova) [inline src/product-switch.scss]: Can we use rem?

File: src/product-switch.scss [hunk 4]
```diff
@@ -148,17 +303,19 @@ $block: #{$fd-namespace}-product-switch;
...
+    }
 
-      .#{$block}__icon,
-      .#{$block}__title,
-      .#{$block}__subtitle {
-        color: var(--sapList_Active_TextColor);
+    .#{$block}__title {
+      &::after,
+      &::before {
+        background-color: var(--sapTile_Background);
```
Comment (InnaAtanasova) [inline src/product-switch.scss]: Do we have this color in List?

File: src/product-switch.scss [hunk 5]
```diff
@@ -88,10 +86,28 @@ $block: #{$fd-namespace}-product-switch;
...
 
       &:focus {
-        outline: none;
-        border: 0.0625rem dotted;
-        border-color: var(--sapContent_FocusColor);
-        border-radius: 0;
+        outline-style: dotted;
+        outline-width: 0.0625rem;
+        outline-color: var(--sapContent_FocusColor);
+      }
```
Comment (InnaAtanasova) [inline src/product-switch.scss]: should be with no outline and border: 0.0625rem dotted

File: src/product-switch.scss [hunk 6]
```diff
@@ -167,20 +324,39 @@ $block: #{$fd-namespace}-product-switch;
...
       }
     }
 
     &:focus {
-      outline: none;
-      border: 0.0625rem dotted;
-      border-color: var(--sapContent_FocusColor);
-      border-radius: var(--sapElement_BorderCornerRadius);
+      outline: 0.0625rem dotted;
+      outline-color: var(--sapContent_FocusColor);
```
Comment (InnaAtanasova) [inline src/product-switch.scss]: should be with no outline and border: 0.0625rem dotted

File: docs/pages/components/product-switch.md (hunk 1 above)
Comment (stefanoScalzo) [reply docs/pages/components/product-switch.md]: nope ill reverse them back thanks
